localgov_base.libraries.yml

```diff
@@ -273,3 +273,9 @@ leaflet:
     theme:
       css/components/leaflet.ie11.css: {}
       css/components/leaflet.css: {}
+
+video:
+  css:
+    theme:
+      css/components/video.ie11.css: {}
```

Comment:
Let's remove this ie11 line. We don't have any ie11 specific video css (I don't think).

After that, merge merge merge!

Reply:
done. I'm hitting the merge button.
I guess we'll need a release at some point?